### PR TITLE
Major updates and reorganizing of checks.mdx

### PR DESCRIPTION
**Pull request by am-ak**
### Description
- Removing Duplication at the top of the document
- Correcting links to go from less detailed section to more detailed section
- Removing misplaced Headlines

### PR Checklist
* [ ] updated test coverage
* [x] external facing docs updated
* [x] not a security concern

### CODE REVIEW

File: website/content/docs/discovery/checks.mdx
Comment (jkirschner-hashicorp) [inline website/content/docs/discovery/checks.mdx:80]: The `((#script-interval))` is an anchor tag alias that preserves backwards compatibility for other content that links to information about this kind of health check. Do we have a need to break that backwards compatibility? If not, I recommend preserving the existing link aliases (for all check types, not just this one).

**Review by jkirschner-hashicorp — CHANGES_REQUESTED** [review]
Thanks for taking this on! I added a blocking review comment about link backwards compatibility that I want to make sure we align on before merging (separate from the education team review).

File: website/content/docs/discovery/checks.mdx
Comment (jkirschner-hashicorp) [reply website/content/docs/discovery/checks.mdx:80]: CC @boruszak

Comment (am-ak) [reply website/content/docs/discovery/checks.mdx:80]: the reason is that in the original doc, the detailed section at the bottom with the examples and all, was a link to the redundant section at the top so for example http check pointed to http+interval. with the update and shortening of the top section it would be even more ridiculous

Comment (jkirschner-hashicorp) [reply website/content/docs/discovery/checks.mdx:80]: I had previously restructured this page and was surprised to find the redundant info at the top. I'm not sure if I accidentally left that in, or if something weird happened in that merge or since. Either way, you're right that the bulleted items at the top were being linked to, not the details below, so we're effectively still preserving the old links with your change.

Cool - I'll dismiss my blocking review!

**Review by jkirschner-hashicorp — COMMENTED** [review]
Dismissing my blocking review. LGTM

**Review by jkirschner-hashicorp — APPROVED** [review]
LGTM - pending the addressing of feedback from @boruszak

**Pull request merged**
